src/models/storage.ts

```diff
@@ -193,77 +207,79 @@ class EntryStorage {
             chrome.storage.sync.get((_data: {[hash: string]: OTPStorage}) => {
               const data: OTPEntry[] = [];
               for (let hash of Object.keys(_data)) {
-                const entryData = _data[hash];
-                let needMigrate = false;
+                if (this.ensureObject(_data, hash)) {
```

Comment:
Same here, we can just skip the loop if check not passes

Reply:
On the loops, I did it like this so that it just ignores the invalid data and keeps looping so that it will continue to process valid data, if I just have it break the loop won't it stop loading as soon as it runs into data it doesn't understand?

Reply:
We do not `break` the loop here, but skip by using `continue` ;-P

Reply:
How did I not know about that >_<